doc/system_requirements.md

```diff
@@ -305,9 +305,124 @@ Needs: dsn
 #### Get Changed Dependency
 `req~npm-changed-dependency~1`
 
-PK can retrieve changed NPM dependencies that where added, updated or removed since the last release.
+PK can retrieve changed NPM dependencies that were added, updated or removed since the last release.
 
 Covers:
 * `feat~npm-project-support~1`
 
 Needs: dsn
+
+### Automatic Dependency Update Process
+`feat~automatic-dependency-update-process~1`
+
+PK supports the automatic dependency process. This speeds up fixing vulnerabilities in third party dependencies and creating releases.
+
+Rationale:
+
+We have 130+ projects in the integration team that often require dependency updates due to security issues that are found in the dependencies or transitive dependencies. Most of the time the update is a combination of pulling the latest source, updating the dependencies, updating the change log, running the tests locally, on success pushing the branch, running CI and creating a release.
+
+* Auto-update dependencies
+* Automatically create change log entry
+* Automatically run local tests
+* Automatically push branch
+* Automatically run CI
+* Release
+
+Needs: req
+
+#### Auto-update dependencies
+`req~auto-update-dependencies~1`
+
+PK automatically updates dependencies when a new vulnerability is found.
+
+Covers:
+* [`feat~automatic-dependency-update-process~1`](#automatic-dependency-update-process)
+
+Needs: dsn
+
+#### Automatically create change log entry
+`req~auto-create-changelog~1`
+
+PK automatically generates the change log for fixed vulnerabilities.
+
+Rationale:
+
+The change log for fixed vulnerabilities always has the same structure and can be easily automated to avoid manual work.
```

Comment:
How does this work, if there is already a changelog file with entries for a new, unreleased version?

Reply:
PK must update the changelog file accordingly. This is the difficult part :-P

Reply:
From terminology perspective I would love to distinguish between 
* changelog = list of revisions
* changes file = details about the changes in a single revision

In case we need an over-arching term for both items, maybe including additional components, maybe being a little bit more abstract, we could think of something like _changes documentation_ or _list of changes_ etc.

Reply:
✅